docs/flash-flood.ipynb

```diff
@@ -0,0 +1,257 @@
+      "new journal 1970-01-06T080640.000000Z--1970-01-06T080640.000000Z--new--0c44793e-1a43-44ca-b013-b4bd155fcf70\n",
+      "replaying from journal 1970-01-05T124000.000000Z--1970-01-05T124000.000000Z--new--9df9d4b4-deb6-4bee-9c8a-00ad846df92a\n",
+      "replaying from journal 1970-01-05T124000.000000Z--1970-01-05T124000.000000Z--new--ee8ef69e-f460-4ba6-b0db-6e4b3dc2ace5\n",
+      "replaying from journal 1970-01-05T152640.000000Z--1970-01-05T152640.000000Z--new--21db50e4-40f9-4ab3-87fa-f89e3edf592b\n",
+      "b'{\"mynum\": 43}'\n",
+      "replaying from journal 1970-01-05T152640.000000Z--1970-01-05T152640.000000Z--new--e15c5bf6-10e5-4eb3-bca3-48a9aef08159\n",
+      "b'{\"mynum\": 43}'\n",
+      "replaying from journal 1970-01-05T181320.000000Z--1970-01-05T181320.000000Z--new--6462df35-648d-41a4-bbc1-793f76f384e9\n",
+      "b'{\"mynum\": 44}'\n",
+      "replaying from journal 1970-01-05T181320.000000Z--1970-01-05T181320.000000Z--new--dc08b579-9eeb-4bf7-95bd-adc3b2062cd0\n",
+      "b'{\"mynum\": 44}'\n",
+      "replaying from journal 1970-01-05T210000.000000Z--1970-01-05T210000.000000Z--new--216c64cf-53b4-433e-8cee-981847e8ccc3\n",
+      "b'{\"mynum\": 45}'\n",
+      "replaying from journal 1970-01-05T210000.000000Z--1970-01-05T210000.000000Z--new--d9419e46-582e-4932-9a91-ac1764940573\n",
+      "b'{\"mynum\": 45}'\n",
+      "replaying from journal 1970-01-05T234640.000000Z--1970-01-05T234640.000000Z--new--315e66a0-cd7b-4119-8402-16024de320f2\n",
+      "b'{\"mynum\": 46}'\n",
+      "replaying from journal 1970-01-05T234640.000000Z--1970-01-05T234640.000000Z--new--f2cd1e7e-c52f-4689-8039-1a13552c5f21\n",
+      "b'{\"mynum\": 46}'\n",
+      "replaying from journal 1970-01-06T023320.000000Z--1970-01-06T023320.000000Z--new--04b9cc76-a129-4153-9b0d-524e3f5d5fcc\n",
+      "b'{\"mynum\": 47}'\n",
+      "replaying from journal 1970-01-06T023320.000000Z--1970-01-06T023320.000000Z--new--d66c6b50-94dd-4784-8be2-c360bab935fa\n",
+      "b'{\"mynum\": 47}'\n",
+      "replaying from journal 1970-01-06T052000.000000Z--1970-01-06T052000.000000Z--new--bbf2269d-4faa-4e63-b71c-35bde72112f8\n",
+      "b'{\"mynum\": 48}'\n",
+      "replaying from journal 1970-01-06T052000.000000Z--1970-01-06T052000.000000Z--new--e590dc6e-a350-4ef6-b833-6e51b98c6908\n",
+      "b'{\"mynum\": 48}'\n"
+     ]
+    }
+   ],
+   "source": [
+    "import json\n",
+    "\n",
+    "for i in range(40, 50):\n",
+    "    event_data = json.dumps({'mynum': i}).encode()\n",
+    "    event_uuid = str(uuid.uuid4())\n",
+    "    event_date = datetime.datetime.fromtimestamp(10000 * i)\n",
+    "    ff.put(event_data, event_uuid, event_date)\n",
+    "\n",
+    "arbitrary_from_date = datetime.datetime.fromtimestamp(10000 * 42)\n",
+    "arbitrary_to_date = datetime.datetime.fromtimestamp(10000 * 48)\n",
+    "\n",
+    "for event in ff.replay(from_date=arbitrary_from_date, to_date=arbitrary_to_date):\n",
+    "    print(event.data)"
```

Comment:
just a nit, could there be an end of line delimiter such as `\n`? makes it easier to see.